ui/lib/store.js

```diff
@@ -46,12 +46,12 @@ const initialState = fromJS({
     grafanaBoardSearch: '',
     grafanaBoards: [],
     selectedBoardsConfigs: [],
-    ts: new Date(),
+    ts: new Date(-8640000000000000),
```

Comment:
@dhruv0000 what is the logic here? Is this to invalidate any cached info?

Reply:
To not let the initialized empty data be seen as the latest data.